darkcluster/src/main/java/com/linkedin/darkcluster/impl/DarkClusterStrategyFactoryImpl.java

```diff
@@ -85,14 +85,11 @@ public DarkClusterStrategyFactoryImpl(@Nonnull Facilities facilities,
   public void start()
   {
     // make sure we're listening to the source cluster and have strategies for any
-    // associated dark clusters. The order matters here: we directly invoke onClusterAdded to
-    // directly query zookeeper for the DarkClusterConfigMap, and then we add the listener.
-    // Doing it in reverse causes a timeout. Ideally, registerClusterListener would have some
-    // functionality similar to SimpleLoadBalancer.listenToCluster that both adds a listener and
-    // requests the data through the eventbus, which will trigger ClusterSubscriber::handlePut and
-    // and iterate through the ClusterListeners.
-    _clusterListener.onClusterAdded(_sourceClusterName);
+    // associated dark clusters. While registering the cluster listener is enough,
+    // we also "warm up" the strategies directly by triggering the clusterListener so that
+    // we retrieve the dark clusters before any inbound request. This is a
```

Comment:
Is this comment got cut-off at the end of the line?

Reply:
oops, will remove that, thanks for catching.